src/wallet/wallet.cpp

```diff
@@ -6826,3 +6826,159 @@ void CWallet::PrecomputeSpends()
         MilliSleep(5000);
     }
 }
+
+boost::thread_group* pthreadGroupAutoSpend;
+
+
+void LinkAutoSpendThreadGroup(void* pthreadgroup)
+{
+    pthreadGroupAutoSpend = (boost::thread_group*)pthreadgroup;
+}
+
+bool StartAutoSpend()
+{
+    if (!pthreadGroupAutoSpend) {
+        error("%s: pthreadGroupAutoSpend is null! Cannot autospend.", __func__);
+        return false;
+    }
+
+    // Close any active precomputing threads before starting new threads
+    if (pthreadGroupAutoSpend->size() > 0) {
```

Comment:
The comments here don't seem to match what this is doing (not precomputing based).

Reply:
Some copy paste errors. Will update comments with correct wording